README.md

```diff
@@ -6,7 +6,19 @@
 
 **Download the binary for your OS**
 
-TODO: add links once releases are built
+- All Mac systems with MacOS 11 (Big Sur) or later, including both Intel and M1 Macs: 
+  [Download here](https://github.com/estuary/flowctl/releases/latest/download/flowctl-multiarch-macos)
```

Comment:
Note that these links won't work until after the first release is published (the pre-release doesn't count).